src/App.css

```diff
@@ -1,3 +1,7 @@
+body {
+  background-color: #6d7c81;
+}
+
 .App {
   text-align: center;
 }
```

Comment:
> Delete all the boilerplate from Create React App (text, images, styles).
- [ ] According to the project requirement, you are meant to delete all Boiulerpalte code from your codebase. I'll suggest you remove all these defaults react css from the codebase. :+1: :+1: